docs/atomic.md

```diff
@@ -0,0 +1,18 @@
+Atomic host bootstrap
+=====================
+
+Atomic host testing has been done with the network plugin flannel. Change the inventory var `kube_network_plugin: flannel`.
```

Comment:
Are we going to test calico and weave? It would be nice to support all networking plugins